dockstore-yaml-proposals/jbrowse/.dockstore.yml

```diff
@@ -0,0 +1,65 @@
+# .dockstore.yml for JBrowse
+version: 1.1
+service:
+  type: docker-compose
+  name: JBrowse
+  author:
+  description: o
```

Comment:
Assuming o is a typo. 
Either while filling in author or as a typo for `|` that I've sometimes seem for multi-line content in yaml.

Reply:
Yes, oops, fixing in next comment; supposed to be `|`.